src/components/OfferDialog.jsx

```diff
@@ -71,13 +76,16 @@ const OfferDialog = ({
   };
 
   const handleConfirm = () => {
-    if (parseFloat(input) < minAmount) {
+    console.log("Dialog input,min,max:", input.toString(), minAmount.toString(),maxAmount.toString());
+
+    if (input.lt(minAmount.toString())) {
```

Comment:
I think minAmount is already a big Number type, so you don't need `.toString()` here